File: atividade 6 c/vendadejogos.c
#include <stdio.h>

int main() {
    int quantidadeJogos;
    printf("Digite a quantidade de jogos vendidos por Catarina: ");
    scanf("%d", &quantidadeJogos);

    double custoPorJogo = 19.90;
    double totalVendas = quantidadeJogos * custoPorJogo;
    double bonus = (quantidadeJogos / 15.0) * 0.08 * totalVendas;
    double salario = 0.5 * totalVendas;
    double totalRecebido = salario + bonus;

    printf("Total de Vendas: %.2f\n", totalVendas);
    printf("Bônus: %.2f\n", bonus);
    printf("Total Recebido: %.2f\n", totalRecebido);

    return 0;
}

File: atividade 6 haskell/vendadejogos.hs
main :: IO ()
main = do
    putStr "Digite a quantidade de jogos vendidos por Catarina: "
    quantidadeJogos <- readLn :: IO Int

    let custoPorJogo = 19.90
        totalVendas = fromIntegral quantidadeJogos * custoPorJogo
        bonus = (fromIntegral quantidadeJogos / 15) * 0.08 * totalVendas
        salario = 0.5 * totalVendas
        totalRecebido = salario + bonus

    putStrLn $ "Total de Vendas: " ++ show totalVendas
    putStrLn $ "Bônus: " ++ show bonus
    putStrLn $ "Total Recebido: " ++ show totalRecebido
